### PR TITLE
[DEV-1066] Add webinar and speaker types

### PR DESCRIPTION
**Pull request by marcobottaro**
#### List of Changes

Add webinar and speaker types

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/lib/types/webinar.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Speaker } from '@/lib/types/speaker';
+
+export type Webinar = {
+  readonly description: string;
+  readonly speakers?: readonly Speaker[];
```
Comment (kin0992) [inline apps/nextjs-website/src/lib/types/webinar.ts:5]: Can a webinar have no speakers?

File: apps/nextjs-website/src/lib/types/webinar.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  readonly startDateTime?: Date;
```
Comment (kin0992) [inline apps/nextjs-website/src/lib/types/webinar.ts:6]: Also the startDateTime shouldn't be required?

File: apps/nextjs-website/src/lib/types/webinar.ts (hunk 1 above)
Comment (marcobottaro) [reply apps/nextjs-website/src/lib/types/webinar.ts:5]: Yes, I imagined the scenario where the speakers are not yet confirmed (and the date is not sure*), but the devPortal's staff needs to start a campaign about the upcoming webinar

File: apps/nextjs-website/src/lib/types/webinar.ts (hunk 2 above)
Comment (marcobottaro) [reply apps/nextjs-website/src/lib/types/webinar.ts:6]: * for the same reason of the undefined speakers

File: apps/nextjs-website/src/lib/types/webinar.ts (hunk 1 above)
Comment (kin0992) [reply apps/nextjs-website/src/lib/types/webinar.ts:5]: yep, makes sense, and an empty array can let you handle webinars without speakers.
if the `speakers` property can also be `undefined` we should both handle `undefined` values and empty arrays

**Review by kin0992 — APPROVED** [review]

**Pull request merged**